cortex-installer.sh

```diff
@@ -1933,15 +1933,16 @@ function prompt_for_usage_stats() {
     while true
     do
       echo
-      read -p "Would you like to help improve Cortex by anonymously sending errors and usage stats to the dev team [Y/n] " -n 1 -r
+      read -p "Would you like to help improve Cortex by anonymously sending errors and usage stats to the dev team [y/n] " -n 1 -r
       echo
-      if [[ $REPLY =~ ^[Yy]$ ]]; then
```

Comment:
We might as well still accept capital letters, since it's safe enough to assume "Y" means yes and "N" means no